wordpress-mobile/gutenberg-mobile
# Issue 331/mention support on @ keypress

Fixes #331

Related PRs:

 - Gutenberg PR: https://github.com/WordPress/gutenberg/pull/22119
 - WP-iOS: https://github.com/wordpress-mobile/WordPress-iOS/pull/14088
 - WP-Android: https://github.com/wordpress-mobile/WordPress-Android/pull/12115

This PR changes the AztecView component to be able to intercept any keycode that is defined. It then configures uses this functionality to intercept the `@` keypress and start the mention UI.

To test:
 - Start the demo app
 - Add a Paragraph block
 - Tap the @ on the keyboard
 - Check if a mention to `matt` is added
 - Check if the mention is only inserted if the @ is inserted with a space before or at the start of the paragraph.

PR submission checklist:
